example/demo.c

```diff
@@ -1060,6 +1066,58 @@ void drawScissor(NVGcontext* vg, float x, float y, float t)
 	nvgRestore(vg);
 }
 
+
+void drawGlobalCompositeOperation(NVGcontext* vg, float x, float y, DemoData* data)
+{
+    int i;
+    NVGpaint imgPaint;
+    enum NVGcompositeOperation ops[] = {
+        NVG_SOURCE_OVER,
+        NVG_SOURCE_IN,
+        NVG_SOURCE_OUT,
+        NVG_ATOP,
+        NVG_DESTINATION_OVER,
+        NVG_DESTINATION_IN,
+        NVG_DESTINATION_OUT,
+        NVG_DESTINATION_ATOP,
+        NVG_LIGHTER,
+        NVG_COPY,
+        NVG_XOR};
+    
+    int count = sizeof(ops)/sizeof(ops[0]);
+
+    
+    for(i=0;i<count;++i)
+    {
+        nvgSave(vg);
+        
+        nvgTranslate(vg, x + i * 40, y);
+        
+        nvgGlobalCompositeOperation(vg, NVG_COPY);
```

Comment:
Thinking you do not merge if this problem is not resolved? I think this PR is not direct related to the problem.